apps/aecore/src/aec_chain_state.erl

```diff
@@ -883,15 +884,21 @@ grant_fees(Node, Trees, Delay, FraudStatus, State) ->
     {BeneficiaryReward1, BeneficiaryReward2, LockAmount} =
         calc_rewards(FraudStatus1, FraudStatus2, KeyFees, MineReward2,
                      FraudReward1, node_is_genesis(KeyNode1, State)),
+
+    OldestBeneficiaryHeight = node_height(KeyNode1),
+    {AdjustedReward1, AdjustedReward2, FoundationReward} =
+        reward_split_for_protocol_foundation(BeneficiaryReward1, BeneficiaryReward2, OldestBeneficiaryHeight),
+
     Trees1 =
         lists:foldl(
             fun({K, Amt}, TreesAccum) when Amt > 0 ->
                     aec_trees:grant_fee(K, TreesAccum, Amt);
                 (_, TreesAccum) -> TreesAccum
             end,
             Trees,
-            [{Beneficiary1, BeneficiaryReward1},
-             {Beneficiary2, BeneficiaryReward2}]),
+            [{Beneficiary1, AdjustedReward1},
+             {Beneficiary2, AdjustedReward2},
```

Comment:
May I suggest we clarify https://github.com/aeternity/protocol/pull/344 before merging this PR.

Reply:
merged